### PR TITLE
Promisify

### PR DESCRIPTION
**Pull request by SamVerschueren**
Hi, I promisified the library so it's not using the callback style anymore. 

Promises all the way ;)!

On a sidenote, I didn't run the tests as I don't work with Windows. Can someone run the tests and let me know :)?

// @kevva @sindresorhus

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -16,13 +16,9 @@ if (!cli.input.length) {
 	process.exit(1);
 }
 
-nircmd(cli.input, function (err, res) {
-	if (err) {
+nircmd(cli.input)
+	.then(console.log)
+	.catch(function (err) {
```
Comment (kevva) [inline cli.js]: `meow` catches this for us.

Comment (SamVerschueren) [inline cli.js]: Oh yes, forgot. Sindre already pointed that out for me :).

File: test.js [hunk 2]
```diff
@@ -1,29 +1,19 @@
...
+import fn from './';
 
-test('clipboard', function (t) {
-	t.plan(1);
-
-	nircmd('clipboard set "hey everybody"', function (err) {
-		t.assert(!err, err);
-	});
+test('clipboard', async t => {
+	await nircmd('clipboard set "hey everybody"');
```
Comment (gillstrom) [inline test.js]: Change `nircmd` to `fn`.

File: test.js [hunk 3]
```diff
@@ -1,29 +1,19 @@
...
 });
 
-test('screensaver', function (t) {
-	t.plan(1);
-
-	nircmd('screensaver', function (err) {
-		t.assert(!err, err);
-	});
+test('screensaver', async t => {
+	await nircmd('screensaver');
```
Comment (gillstrom) [inline test.js]: Change `nircmd`to `fn`.

File: test.js [hunk 4]
```diff
@@ -1,29 +1,19 @@
...
 });
 
-test('screensaver using spawn', function (t) {
-	t.plan(1);
-
-	var cp = nircmd.spawn('screensaver');
+test('screensaver using spawn', t => {
+	const cp = nircmd.spawn('screensaver');
```
Comment (gillstrom) [inline test.js]: Change `nircmd` to `fn`.

File: test.js [hunk 5]
```diff
@@ -1,29 +1,19 @@
...
 
 	cp.on('exit', function (code) {
-		t.assert(code === 0);
+		t.true(code === 0);
+		t.end();
```
Comment (kevva) [inline test.js]: Use `t.plan()` instead of `t.end()` here.

Comment (SamVerschueren) [inline test.js]: Is it better to use `t.plan()` then?

Comment (kevva) [inline test.js]: Doesn't matter that much, but feels like a good practice to define planned assertions.

**Comment by kevva** [comment]
lgtm :). Great work. Could you squash ze commits?

**Comment by gillstrom** [comment]
Looks good! :stars:

**Comment by SamVerschueren** [comment]
Could I do what :p?

**Comment by kevva** [comment]
Squash them together so they are two commits instead of four, `git rebase -i HEAD~4`.

Mark `fixed readme` and `meow catches...` with `f` so that they are squashed into `Promisified library` :).

**Comment by SamVerschueren** [comment]
Alright, learning some new things here :). On it!

**Comment by SamVerschueren** [comment]
Not sure, but did I just screwed things up?

**Comment by kevva** [comment]
Nope, that's all good :D. You could even include that last commit too. I thought you had splitted the "promisifying" and "ES2015ifying" of the tests up, but since you haven't you could squash the commits into one :).

**Comment by SamVerschueren** [comment]
Ok, will do that :)

**Comment by SamVerschueren** [comment]
That should be it! Thanks for helping me out on this and for the feedback :).

**Comment by gillstrom** [comment]
Thanks for promisfying!

**Pull request merged**